### PR TITLE
Drop old components installation method

### PR DESCRIPTION
**Pull request by invidian**
We now install all components using helm underneath, so support for old installation method can be now removed.

### CODE REVIEW

**Comment by invidian** [comment]
>CI is failing.

Oops, I didn't clean the unit tests. Fixed now.

**Comment by iaguis** [comment]
It seems openebs-operator can't be installed through helm?

**Comment by invidian** [comment]
>It seems openebs-operator can't be installed through helm?

I didn't really test this code, so fixing now :see_no_evil:

File: pkg/components/util/install.go [hunk 1]
```diff
@@ -122,17 +102,13 @@ func InstallAsRelease(name string, c components.Component, kubeconfig string) er
 		// The example of such dependency is between prometheus-operator and openebs-storage-class, where
 		// both openebs-operator and openebs-storage-class components must be fully functional, before
 		// prometheus-operator is deployed, otherwise it won't pick the default storage class.
-		install.Wait = c.Metadata().Helm.Wait
-
-		if _, err := install.Run(chart, map[string]interface{}{}); err != nil {
```
Comment (invidian) [inline pkg/components/util/install.go]: Guess why it doesn't install components anymore... :smile:

Comment (invidian) [reply pkg/components/util/install.go]: Fixed now. I also managed to fix checking for nil pointers.

**Pull request merged**